.azure/ipu-tests.yml

```diff
@@ -76,7 +76,8 @@ jobs:
         for fpath in `ls requirements/**/*.txt`; do \
           python ./requirements/pytorch/adjust-versions.py $fpath; \
         done
-        pip install -e .[dev]
+        pip install -e .[extra,examples,test]
```

Comment:
Looks like we should also avoid installing the strategies for the tpu tests

Reply:
I knew all this is a trap...
There is no indication whatsoever in the tpu-tests.yml that we are installing anything.

Reply:
It's here: https://github.com/Lightning-AI/lightning/blob/fbbbbf6ad7aef07f418df198ffde26276419fdd6/dockers/base-xla/tpu_workflow_pytorch.jsonnet#L45